.github/workflows/test.yml

```diff
@@ -22,3 +22,28 @@ jobs:
         uses: gradle/wrapper-validation-action@v1
       - name: Gradle Run Tests
         run: ./gradlew library:testDebug
+  static-analysis:
```

Comment:
I could be down to moving this into a new workflow file called `lint.yml` instead of adding it to test. Thoughts?

Reply:
Up to you IMO! Both work for setting up the gh rule I believe so just personal pref

Reply:
I decided to move it. Since we will add the example app tests in this file as well.